### PR TITLE
Restore: First stage of access logging middleware.

### PR DESCRIPTION
**Pull request by ldez**
### Description

Restore access logging middleware.

Related to #1541, #1408, #1485

### CODE REVIEW

File: middlewares/accesslog/save_backend.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+func (sb *SaveFrontend) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	table := GetLogDataTable(r)
+	table.Core[FrontendName] = strings.TrimPrefix(sb.frontendName, "frontend-")
+
+	sb.next.ServeHTTP(rw, r)
+}
+
+//-------------------------------------------------------------------------------------------------
+// the next 3 function (SaveNegroniFrontend, NewSaveNegroniFrontend, ServeHTTP) are temporary,
```
Comment (ldez) [inline middlewares/accesslog/save_backend.go]: Because the code evolve between the first merge and the revert restoration.

Comment (timoreimann) [inline middlewares/accesslog/save_backend.go]: Okay, I realized now that the functions _were_ in the original PR but didn't include the `Negroni` part in the name.

Why the warning at the top and why change the names?

Comment (emilevauge) [inline middlewares/accesslog/save_backend.go]: @timoreimann we worked both on resolving this conflict. we had to do this to manage different types negroni and http handlers.

Comment (timoreimann) [inline middlewares/accesslog/save_backend.go]: 👍

**Pull request merged**